app/src/main/java/com/lagradost/cloudstream3/ui/player/AbstractPlayerFragment.kt

```diff
@@ -622,9 +630,20 @@ abstract class AbstractPlayerFragment(
             PlayerResize.Fill -> AspectRatioFrameLayout.RESIZE_MODE_FILL
             PlayerResize.Fit -> AspectRatioFrameLayout.RESIZE_MODE_FIT
             PlayerResize.Zoom -> AspectRatioFrameLayout.RESIZE_MODE_ZOOM
+            PlayerResize.SixteenByNine -> 16.0f/9.0f
+            PlayerResize.NineBySixteen -> 9.0f/16.0f
+            PlayerResize.OneByOne -> 1.0f/1.0f
+            PlayerResize.IMAX -> 1.43f / 1.0f
+            PlayerResize.DolbyVision -> 2.39f / 1.0f
+            PlayerResize.UltraWide -> 21.0f / 9.0f
+            PlayerResize.Expanded -> 1.90f / 1.0f
+        }
+        if(type is Float){
+            playerView?.resizeMode = AspectRatioFrameLayout.RESIZE_MODE_FIT
+            aspectRatioFrameLayout?.setAspectRatio(type)
+        }else {
+            playerView?.resizeMode = type as Int
```

Comment:
Avoid unchecked casts in Android, they will crash if not satisfied. Use `else if (type is Int)` for the exact same effect, but more future proof

Reply:
Yea, this is correct. I have Updated this code in my local